lib/index.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const path = require('path');
+const querystring = require('querystring');
+const fetch = require('node-fetch');
```

Comment:
If it weren't already guaranteed to be in the dependency tree already then I'd possibly agree, but it's already there, and encapsulating the functionality also allows for it to be more easily customisable by an end user.